README.md

```diff
@@ -1,5 +1,24 @@
 # kubernetes-credentials-provider-plugin
 Credentials provider for Kubernetes
 
-## Examples
+## Pre-requisites
+
+- Jenkins must be running in a kubernetes cluster
+- The pod running Jenkins must have a service account with a role that sets the following:
+  - get/watch/list permissions for *configmaps* and *secrets*
```

Comment:
you should not need any permission on `configmap`s as the code only calls the secrets API.
that said I didn;t look what was set on the user (I just added permissions to secrets and it may have had configmap access already that is somehow needed - but can you confirm that you did have to do this)?

Reply:
NIT: better to use backtick than * for code / api relating things.

Reply:
The error message in the console was indicating that configmap permissions were required

Reply:
Hmm....  I shall double check that locally.  What engine are you testing on?  I am using GKE.

Reply:
@cwebbtw could you review #6 (esp [testing/secret-reader-role.yaml](https://github.com/jenkinsci/kubernetes-credentials-provider-plugin/pull/6/files#diff-a892866445db34f517c5754592a34821)) I have only granted get/watch/list on secrets.

Reply:
Using AWS - reviewed,

Thanks!